src/cayenne/tasks/category.clj

```diff
@@ -76,7 +76,7 @@
                         (sheet/select-sheet categories-sheet-name)
                         (sheet/select-columns {:A :code :B :name})
                         (drop 1)
-                        (filter #(-> % :code nil? not)))]
+                        (keep :code))]
```

Comment:
Outstanding question about behaviour of keep in this context. Wouldn't it return the value of `:code` (i.e. like `(map :code)`) rather than simply filter?

Reply:
Yeah, this looks like a mistake, I will correct it.